src/coreclr/src/System.Private.CoreLib/src/System/Threading/ThreadPool.CoreCLR.cs

```diff
@@ -18,6 +18,12 @@
 
 namespace System.Threading
 {
+    internal static partial class ThreadPoolGlobals
+    {
+        public static bool enableWorkerTracking;
+        public static bool ThreadPoolInitialized { get; } = ThreadPool.InitializeThreadPool();
+    }
```

Comment:
We can't just have these on ThreadPool itself and avoid the need for EnsureInitialized() completely?

Reply:
Yeah, that is better :)